src/Parse/EnvFile.php

```diff
@@ -0,0 +1,238 @@
+<?php namespace  Winter\Storm\Parse;
+
+use Winter\Storm\Support\Str;
+use Winter\Storm\Parse\Contracts\DataFileInterface;
+
+/**
+ * Class EnvFile
+ */
+class EnvFile implements DataFileInterface
+{
+    /**
+     * @var array Lines of env data
+     */
+    protected $env = [];
+
+    /**
+     * @var array Map of variable names to line indexes
+     */
+    protected $map = [];
+
+    /**
+     * @var string|null Filepath currently being worked on
+     */
+    protected $filePath = null;
```

Comment:
@jaxwilko Since we're targeting PHP 8, you could just put a type hint for these properties :)